integration_tests/run_e2e_tests.py

```diff
@@ -309,7 +329,7 @@ def e2e_tests(
 def print_failed_test_results(e2e_target: str, failed_test_results: List[TestResult]):
     print(f"Failed {e2e_target} tests:")
     for failed_test_result in failed_test_results:
-        print(f"{failed_test_result.type}: {failed_test_result.message}")
+        print(f"\033[1m\033[91m{failed_test_result.type}: {failed_test_result.message}\033[0m")
```

Comment:
Wanted the end result to be bold and more noticeable, I can remove if it you think it's ugly